automation/check-patch.e2e-okd.sh

```diff
@@ -7,15 +7,12 @@
 # yum -y install automation/check-patch.packages
 # automation/check-patch.e2e-okd.sh
 
-# FIXME: Delete this when okd lane works
-exit 0
-
 teardown() {
     make cluster-down
 }
 
 main() {
-    export KUBEVIRT_PROVIDER='okd-4.1'
+    export KUBEVIRT_PROVIDER='ocp-4.3'
```

Comment:
hmm dont we need quay login here ?

Reply:
You are right, will send a PR in a moment.